src/components/common/navbar/Navbar.js

```diff
@@ -45,54 +45,88 @@ class Navbar extends Component {
     } = this.props
 
     return (
-      <Grid.Row className="noPadding">
+      <div className="navbar">
         {head}
 
-        <Grid.Column className="navbar noPadding">
-          <Menu borderless as="nav">
-            <Menu.Menu className={search ? 'sideAreaWithSearch' : 'sideAreaWithoutSearch'}>
-              <Menu.Item icon active={sidebarVisible} name="sidebar" onClick={handleSidebarToggle}>
-                <Icon name="sidebar" />
-              </Menu.Item>
-              <Menu.Header as="h1" className="navbarTitle" content={title} />
-            </Menu.Menu>
+        <div className="sideArea">
+          <Menu borderless className="noBorder">
+            <Menu.Item icon active={sidebarVisible} name="sidebar" onClick={handleSidebarToggle}>
+              <Icon name="sidebar" />
+            </Menu.Item>
+            <h1>
+              {title}
+            </h1>
+          </Menu>
+        </div>
 
-            {search && <SearchArea intl={intl} handleSearch={search.handleSearch} />}
+        {search &&
+          <div className="searchArea">
+            <SearchArea intl={intl} handleSearch={search.handleSearch} />
+          </div>}
 
-            <Menu.Menu className={search ? 'sideAreaWithSearch' : 'sideAreaWithoutSearch'}>
-              <Menu.Menu position="right">
-                <SessionArea sessionId={accountShort} />
-                <AccountArea accountShort={accountShort} />
-              </Menu.Menu>
+        <div className="accountArea">
+          <Menu borderless className="noBorder">
+            <Menu.Menu position="right">
+              <SessionArea sessionId={accountShort} />
+              <AccountArea accountShort={accountShort} />
             </Menu.Menu>
           </Menu>
+        </div>
 
-          <style jsx global>{`
-            .navbarTitle {
-              font-size: 1.3rem;
-              margin-left: 1rem;
-            }
+        <style jsx>{`
+          .navbar {
+            display: flex;
+            align-items: center;
+            flex-flow: row wrap;
+            justify-content: space-between;
+
```

Comment:
On purpose!

@pascalzehnder I'm leaving a space between rules concerning flexbox and other rules.